lib/assets/blog-article.mst

```diff
@@ -5,9 +5,9 @@
 </head>
 
 <body>
-<header class="extra-header">
```

Comment:
Thought `extra-header` and `extra-header-right` don't really define the information very well and seems too specific. If a designer were looking at this, they'd be prepared to look for something having to do with "context" and "support", etc.